hk.yaml

```diff
@@ -43,16 +23,58 @@ months:
     regions: [hk]
     mday: 1
     observed: to_monday_if_sunday(date)
+  - name: Lunar New Year's Day
+    regions: [hk]
+    function: lunar_to_solar(year, month, day, region)
+    mday: 1
+    observed: to_monday_if_sunday(date)
+  - name: The second day of Lunar New Year
+    regions: [hk]
+    function: lunar_to_solar(year, month, day, region)
+    mday: 1
+    observed: to_the_weekday_after(date)
+  - name: The third day of Lunar New Year
+    regions: [hk]
+    function: lunar_to_solar(year, month, day, region)
+    mday: 1
+    observed: to_the_second_weekday_after(date)
+  4:
+  - name: Ching Ming Festival
+    regions: [hk]
+    mday: 5
+    observed: to_previous_day_if_leap_year(date)
```

Comment:
Ching Ming's date is not according to the leap year, it's one of the 24 solar terms (節氣). For example, 2021's Ching Ming is also 4/4

Reply:
Thanks! I'll fix asap

Reply:
Sorry for the late reply.

According to wikipedia (https://en.wikipedia.org/wiki/Public_holidays_in_Hong_Kong), the definition of the Ching Ming festival is `5 April (4 April in leap years), Occurs about 15 days after the March Equinox`.

In 2021 (not a leap year), the _observed day_ is April 5th (The day following Ching Ming Festival) (https://www.gov.hk/en/about/abouthk/holiday/2021.htm)

The actual day is indeed April 4th, but since we are only interested in when the holidays is observed, I'll keep the wikipedia formula that is much simpler than the real one (I actually wasn't able to find the formula for the real calculation).

If you can provide a source for the actual formula and if it's not too complicated I might be able to add it to this PR, but in practice the current code should work.